vm/actor/src/builtin/init/mod.rs

```diff
@@ -90,15 +90,15 @@ impl Actor {
         rt.create_actor(&params.code_cid, &id_address)?;
 
         // Invoke constructor
+
         rt.send(
             &id_address,
-            METHOD_CONSTRUCTOR,
+            1,
             &params.constructor_params,
             &rt.message().value().clone(),
         )
-        .map_err(|err| rt.abort(err.exit_code(), "constructor failed"))?;
-
-        Ok(ExecReturn {
+        .map_err(|err| rt.abort(err.exit_code(), "constructor failed"))
+        .map(|_| ExecReturn {
```

Comment:
I mean, this works exactly the same but is a bit confusing because you are using map but not mapping any values. This is the same as doing a try return `?` and just returning the Ok `ExecReturn`